js/src/data/reducer.js

```diff
@@ -306,6 +311,15 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 			);
 		}
 
+		case TYPES.RECEIVE_MC_REVIEW_REQUEST: {
+			const { issues, nextAttempt, status } = action.mcReviewRequest;
+			return chainState( state, 'mc_review_request' )
+				.setIn( 'issues', issues )
+				.setIn( 'nextAttempt', nextAttempt )
+				.setIn( 'status', status )
+				.end();
```

Comment:
💅  + ❔ 

Looks like it could be replaced with below?

```js
return setIn( state, 'mc_review_request', action.mcReviewRequest );`
```

Reply:
Thanks. Fixed here [c4fb768](https://github.com/woocommerce/google-listings-and-ads/pull/1234/commits/c4fb76804aa318449a6f87ff7e39f38e2ed81bb5)